graylog2-server/src/test/java/org/graylog2/security/hashing/BCryptPasswordAlgorithmTest.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.security.hashing;
+
+import org.graylog2.Configuration;
+import org.graylog2.security.hashing.BCryptPasswordAlgorithm;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class BCryptPasswordAlgorithmTest {
+    private BCryptPasswordAlgorithm bCryptPasswordAlgorithm;
+
+    @Before
+    public void setUp() throws Exception {
+        final Configuration configuration = new Configuration();
+        this.bCryptPasswordAlgorithm = new BCryptPasswordAlgorithm(configuration.getUserPasswordBCryptSaltSize());
```

Comment:
This will unnecessarily break the test if we change the default hash size. Using `Configuration` here doesn't add anything to the test at all (please correct me, if I overlooked something).